plexapi/base.py

```diff
@@ -231,10 +337,8 @@ def fetchItems(self, ekey, cls=None, container_start=None, container_size=None,
                     fetchItem(ekey, Media__Part__file__startswith="D:\\Movies")
 
         """
-        if ekey is None:
-            raise BadRequest('ekey was not provided')
```

Comment:
Why is guard clause removed?

Reply:
```py
def fetchItems(
        self,
        ekey: Union[str, List[int]],
        cls: Optional[Type[PlexObjectT]] = None,
        container_start: Optional[int] = None,
        container_size: Optional[int] = None,
        maxresults: Optional[int] = None,
        **kwargs: Any,
    ):
```

`ekey` is never `None` as defined by the method, hence unnecessary check. could typehint to accept `None` but then calling `fetchItems` without passing `ekey` does not make sense for the method, `None` should be handled by the caller imo